### ISSUE TITLE
How to set space between two tab

### ISSUE THREAD

**Issue by jabri17** [GocePetrovski/ScrollableSegmentedControl#24]

**Comment by GocePetrovski** [comment]
I don't think we need space between the tabs. Please give me examples on how this would look like.

**Comment by Manoojkumar** [comment]
![Screenshot 2019-09-09 at 3 39 16 PM](https://user-images.githubusercontent.com/15548513/64522770-62b1b500-d318-11e9-812a-e4eccc010a08.png)
Look like this..
How to set ?
